### DISCUSSION TITLE
DivestOS 20

### DISCUSSION THREAD

**Discussion by edern-appere** [Divested-Mobile/DivestOS-Build#182] [General] [answered]
Hi Tad !

I just wanted to ask a couple of things : 

- are DivestOS 20 builds based on official LOS 20 builds released lately (starting january 2023 I think) ?
- are the installation instructions the same as before (#108 ) ?
- I've heard Shelter does not work in LOS 20... is it the same in DivestOS ? is there a workaround to get a work profile working (for app isolation) ?

Thanks for the informations !

Edern

**Comment by SkewedZeppelin** [comment, marked as answer]
DivestOS isn't based on any binary release of LineageOS and is compiled from source.

DivestOS 20.0 has been available since October.

Installation steps are here: https://divestos.org/index.php?page=bootloader#fastboot

Shelter works just fine on DivestOS.

**Reply by edern-appere** [reply]
Thanks for the immediate reply ! 
The sentence on the homepage "DivestOS is a vastly diverged unofficial soft fork of LineageOS." is ambiguous if there is nothing to do with Lineage.

**Reply by SkewedZeppelin** [reply]
It is a fork of their code, and the DivestOS changes are applied directly on their latest changes. But it isn't modifying their binaries or have to track their releases.
